src/GraphQl/Type/SchemaBuilder.php

```diff
@@ -358,6 +360,19 @@ private function convertType(Type $type, bool $input = false, string $mutationNa
                     break;
                 }
 
+                if (is_a($typeClass = $type->getClassName(), Enum::class, true)) {
+                    $typeName = $typeClass::getName();
```

Comment:
Maybe you should add: `/** @var Enum $typeClass */` before to allow IDEs to understand and autocomplete the calls.